crates/client/rpc-core/src/tests.rs

```diff
@@ -78,9 +80,10 @@ fn test_try_into_declare_transaction_v1_max_signature() {
         contract_class: Arc::new(compressed_contract_class),
         sender_address: FieldElement::default(),
     };
-
+    
+    let chain_id = Felt252Wrapper(FieldElement::from_byte_slice_be(b"SN_GOERLI").unwrap());
     let input: BroadcastedDeclareTransaction = BroadcastedDeclareTransaction::V1(txn);
-    let output_result: Result<DeclareTransaction, _> = to_declare_transaction(input);
+    let output_result: Result<DeclareTransaction, _> = to_declare_transaction(input, chain_id);
```

Comment:
This feels like an old version of madara we now use the `From` `TryFrom` trait to be able to directly use `into` or `try_into`. If we don't do that for this type then please add it.